src/stt/create_speech_data.py

```diff
@@ -0,0 +1,184 @@
+# -*- coding: utf-8 -*-
+
+"""
+Description:
+    Creation of the speech data by splitting the recordings
+    into train, validation and test sets.
+    Filtering out recordings that are too long.
+
+Usage:
+    $ python src/data/create_speech_data.py -s
+
+Possible arguments:
+    * -s or --save: Save dataframe to csv
+"""
+import os
+import shutil
+import sys
+
+import click
+import librosa
+import pandas as pd
+from constants import (
+    MAX_DURATION_LENGTH,
+    RAW_DATA_DIR,
+    RAW_RECORDINGS_DIR,
+    RECORDINGS_FILE,
+    SRC_DIR,
+)
+from sklearn.model_selection import train_test_split
+
+sys.path.insert(0, SRC_DIR)
+from decorators import log_function_name  # noqa: E402
+
+
+def get_wav_file_duration(file_path: str) -> float:
+    """
+    Get the duration of a wav file
+
+    Parameters
+    ----------
+    file_path : str
+        Path to the wav file
+
+    Returns
+    -------
+    float
+        Duration of the wav file in seconds
+    """
+    return librosa.get_duration(
+        filename=os.path.join(RAW_RECORDINGS_DIR, file_path)
+    )
+
+
+@log_function_name
+def copy_files(files_list: list[str], folder: str) -> None:
```

Comment:
why do you need to copy files

Reply:
Dann habe ich sie immer an beiden Orten für den Fall das ich etwas an der Funktion verändern will